lib/browser/mqtt.ts

```diff
@@ -115,8 +115,8 @@ export interface MqttConnectionConfig {
     /** Options for the underlying websocket connection */
     websocket?: WebsocketOptions;
 
-    /** AWS credentials, which will be used to sign the websocket request */
-    credentials?: AWSCredentials;
+    /** Options for the underlying credentianls provider */
```

Comment:
This is still a breaking change.  We need to support both possibilities.  If they specify credentials, then use them to make a provider internally, otherwise use the provider.